MediaGalleryMetadataApi/Model/ExtractMetadataComposite.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\MediaGalleryMetadataApi\Model;
+
+use Magento\Framework\Exception\LocalizedException;
+use Magento\MediaGalleryMetadataApi\Api\Data\MetadataInterface;
+use Magento\MediaGalleryMetadataApi\Api\ExtractMetadataInterface;
+
+/**
+ * Metadata extractor composite
+ */
+class ExtractMetadataComposite
```

Comment:
If I understand the approach correctly
```suggestion
class ExtractMetadataComposite implements ExtractMetadataInterface
```